src/database/DataTypes/IndicatorAttribute.js

```diff
@@ -51,5 +60,6 @@ IndicatorAttribute.schema = {
     valueDefault: { type: 'string', default: 'string' },
     axis: { type: 'string', default: 'row' },
     isActive: { type: 'bool', default: false },
+    values: { type: 'list', objectType: 'IndicatorValue' },
```

Comment:
I believe you can do:

```
rowValues: {type: "linkingObjects", objectType: "IndicatorValue", property: "row" }
columnValues: {type: "linkingObjects", objectType: "IndicatorValue", property: "column" }
```

I dunno if that is good or bad but it is something you could do

Reply:
Yeah, I thought about that a bit. Positive is it keeps properties in sync. The only problem is that as each attribute is only a row/column, you'd always have to query both `rowValues` and `columnValues`. My initial thought would be to use this + add a little helper getter, e.g. `get values` which just wraps both properties.